lib/elixir/src/elixir_exp.erl

```diff
@@ -46,6 +46,15 @@ expand({'__aliases__', _, _} = Alias, E) ->
 
 %% alias
 
+expand({Kind, Meta, [{{'.', _, [Base, '{}' | Rest]}, _, Refs}]}, E)
+    when Kind == alias; Kind == require; Kind == import ->
+  if
+    Rest == [] ->
```

Comment:
I believe `Rest` here will be always empty. The Rest with options is in the outer most list. Here is a test that should pass:

    import Foo.{Bar, Baz}, only: :functions